components/common-data-items.js

```diff
@@ -183,7 +183,7 @@ export function SampleDataItems({
       {sortedFractions?.length > 0 && (
         <>
           <DataItemLabel>
-            Sorted Fractions of Sample{sortedFractions.length === 1 ? "" : "s"}
+            Sorted Fraction of Sample{sortedFractions.length === 1 ? "" : "s"}
```

Comment:
I think this should be "Sorted Fraction Sample{sortedFractions.length === 1 ? "" : "s"}"

Reply:
@ian-whaling your comment gave me the option of either so I picked one `I think the title should be renamed to “Sorted Fraction Sample(s)” or “Sorted Fraction(s) of Sample”.`

Reply:
or "Sorted Fraction{sortedFractions.length === 1 ? "" : "s"} of Sample"

Reply:
talked to Ian offline and we both agreed to `Sorted Fraction{sortedFractions.length === 1 ? "" : "s"} of Sample`